ct/datasource_ct_config_test.go

```diff
@@ -191,6 +191,31 @@ const containerLinuxSnippetsExpected = `{
   }
 }`
 
+const containerLinuxSnippetsPrettyFalseResource = `
```

Comment:
Instead of creating entirely separate test configs, I could also convert `containerLinuxSnippetsResource` et al into functions that take a `pretty` argument and use `fmt.Sprintf` to render the configuration. Wanted to avoid modifying other test cases initially but happy to do that now if you'd prefer.

Reply:
I think its fine. Eventually all this custom merging should be handled upstream in Ignition, which would avoid all the branches motivating this exhaustive testing. https://github.com/coreos/fcct/pull/120